archive.php

```diff
@@ -11,20 +10,18 @@
 	{
 		$year = (int)$_GET['year'];
 	}
-	$result = mysql_query("SELECT timestamp, id, title FROM php_blog WHERE FROM_UNIXTIME(timestamp, '%Y') = '$year' ORDER BY id DESC");
+	$result = $database->query("SELECT `timestamp`, `id`, `title` FROM `php_blog` WHERE FROM_UNIXTIME(`timestamp`, '%Y') = '$year' ORDER BY id DESC");
```

Comment:
An example of how to use PDO rather than mysql. the backticks are unrelated to PDO and just good practice when writing SQL